.github/workflows/rebuild_c_files.yml

```diff
@@ -0,0 +1,26 @@
+on:
+  schedule:
+    - cron: 0 0 * * *
```

Comment:
Okay I'm still not sure how running this once a day could consume any significant amount of resources, but you seem pretty adamant about it, and it's a pretty small thing for me. If it's ever annoying that it doesn't run regularly I can always add it back, and it probably won't be.
```suggestion
```

Reply:
I was particularly concerned about polling daily while waiting on an event that probably has a far lower cadence than daily. The concern started when I saw "_python_ version" when it was intended to say "_cython_ version". I was assuming only minor version changes would matter and those only happen ~yearly for Python. I'm not sure about the release cadence of Cython, so can't say for sure, but I'm guessing daily is still rather overkill. I might be able to get onboard with a far lower polling frequency such as 0.5x-1x the typical release cadence of new Cython versions (guessing monthly might be reasonable).

Reply:
Okay, I think once a month should be plenty given our infrequent PRs. I'll add that back.